files/pt-br/learn/forms/form_validation/index.md

```diff
@@ -166,42 +166,42 @@ input:valid {
 
 {{EmbedLiveSample("Validating_against_a_regular_expression", "100%", 80)}}
 
-In this example, the {{HTMLElement("input")}} element accepts one of two possible values: the string "banana" or the string "cherry".
+Neste exemplo, o elemento {{HTMLElement("input")}} aceita um dos dois valores possíveis: a string "banana" ou a string "cherry".
 
-At this point, try changing the value inside the `pattern` attribute to equal some of the examples you saw earlier, and look at how that affects the values you can enter to make the input value valid. Try writing some of your own, and see how you get on! Try to make them fruit-related where possible, so your examples make sense!
+Neste ponto, tente alterar o valor dentro do atributo `pattern` para igualar alguns dos exemplos que você viu anteriormente e veja como isso afeta os valores que você pode inserir para tornar o valor de entrada válido. Tente escrever alguns dos seus próprios, e veja como você se sai! Tente torná-los relacionados a frutas sempre que possível, para que seus exemplos façam sentido!
 
-> **Nota:** Some {{HTMLElement("input")}} element types do not need a {{htmlattrxref("pattern","input")}} attribute to be validated. Specifying the `email` type for example validates the inputted value against a regular expression matching a well-formed email address (or a comma-separated list of email addresses if it has the {{htmlattrxref("multiple","input")}} attribute). As a further example, fields with the `url` type automatically require a properly-formed URL.
+> **Nota:** Alguns tipos de elemento {{HTMLElement("input")}} não precisam de um atributo {{htmlattrxref("pattern","input")}} para serem validados. Especificar o tipo `email`, por exemplo, valida o valor inserido em relação a uma expressão regular que corresponde a um endereço de e-mail bem formado (ou uma lista de endereços de e-mail separados por vírgula se tiver o {{htmlattrxref("multiple","input")} } atributo). Como outro exemplo, os campos com o tipo `url` requerem automaticamente um URL devidamente formado.
```

Comment:
may be this nit, after fixing this, I can build the document on my computer:

```suggestion
> **Nota:** Alguns tipos de elemento {{HTMLElement("input")}} não precisam de um atributo {{htmlattrxref("pattern","input")}} para serem validados. Especificar o tipo `email`, por exemplo, valida o valor inserido em relação a uma expressão regular que corresponde a um endereço de e-mail bem formado (ou uma lista de endereços de e-mail separados por vírgula se tiver o {{htmlattrxref("multiple","input")}} atributo). Como outro exemplo, os campos com o tipo `url` requerem automaticamente um URL devidamente formado.
```

Reply:
yes that's the one @josielrocha! 
Thanks @yin1999 :) 

I don't know why https://github.com/mdn/translated-content/actions/runs/3238711717/jobs/5307264130#step:8:150 complains about line 136 while its on line 173 (or 167 if we remove frontmatter :/)

Reply:
Done.
Thank you for your help!